plugins/python-build/share/python-build/pyston-2.3.1

```diff
@@ -0,0 +1,19 @@
+echo
+colorize 1 "WARNING"
+echo ": Pyston only runs on x86_64 platforms, and only has been tested on Ubuntu"
+echo "so, your mileage may vary with other Linux distributions."
+echo
```

Comment:
There's `require_distro()` to display a warning like this. No need to display it unless the user runs an unsupported distribution.
An an unsupported architecture case is already handled below.

Reply:
I copied this warning from the other pyston rules but I'm happy to change all of them if you'd like.

Reply:
Displaying irrelevant warnings is certainly poor UX.

Reply:
This isn't my project but I have a strong inclination that all of the Pyston installers should have the same behavior. Are you sure that you'd like this one to be different and use require_distro?